src/components/opinion/opinionArticle.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import HeaderImage from '../shared/headerImage';
+import ArticleSeries from '../news/articleSeries';
```

Comment:
What's the difference between `news` and `shared`?

Reply:
I can move some more `news` components to be `shared` actually. 
I was thinking `localhost:8080` is news articles and `http://localhost:8080/commentisfree/2019/dec/09/wins-tories-future-boris-johnson-victory-conservatism` to be Opinion.


So the components used in `news` and `opinion` articles should live in `shared`?

Reply:
Yeah I think if you're going to explicitly specify a `shared` directory then probably everything that's shared should live in there.

Alternatively you could flatten the directory structure and:

a) Have everything shared live directly under `components`, with subdirectories for stuff that isn't shared:

```sh
components/
  headerImage.tsx
  articleSeries.tsx
  opinion/
    opinionArticle.tsx
  ...
```
b) Have everything live directly under `components`:

```sh
components/
  headerImage.tsx
  articleSeries.tsx
  opinionArticle.tsx
```

Reply:
I like the `a` structure, I'm worried that we would have too many shared components at that top level though... which is also a good thing I guess

Reply:
Do you foresee problems with having many components at the top level?

Reply:
Might be harder for new devs or us to find components? Will try moving more into the shared folder now but can refactor later if needed.

Reply:
Yeah sounds good.

On the point about finding components I think you could make the same argument about subdirectories. A path to find a component that seems obvious to one dev may not be obvious to another.

Reply:
Yeah true, we might need to think about where the tests live too because they seem to be bloating folders too at the moment.